mate-extra/mate-sensors-applet/files/1.26.0-clang-16-fix-undeclared-function-setlocale.patch

```diff
@@ -0,0 +1,121 @@
+# unconditionally include stdio.h and locale.h
```

Comment:
Can't you include the commit headers for this? i.e. go to the commit link, append .patch, paste it here, then include your info like you did here at the top?

Reply:
Right. I did .diff hence the missing header info. Fixing